src/common/util/random.js

```diff
@@ -0,0 +1,113 @@
+/*globals define*/
+/*jshint node: true, browser: true, bitwise: false*/
+
+/**
+ * @author kecso / https://github.com/kecso
+ *
+ * collection of functions that uses random Numbers in WebGME
+ */
+
+define(['./chance.min'], function (ChanceJs) {
+    'use strict';
+
+    function random() {
+        var guidPool = '0123456789abcdef',
+            relidPool = '0123456789qwertyuiopasdfghjklzxcvbnmQWERTYUIOPASDFGHJKLZXCVBNM',
+            excludeList = ['atr', 'reg', 'ovr'],
+            maxTry = 2,
+            chance = new ChanceJs(),
```

Comment:
Are you sure chance should be initialized? It should suffice to remove this and just require it as chance.